spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerBusAutoConfiguration.java

```diff
@@ -126,6 +125,7 @@ public AxonServerQueryBus queryBus(AxonServerConnectionManager axonServerConnect
                                  .priorityCalculator(priorityCalculator)
                                  .targetContextResolver(targetContextResolver)
                                  .spanFactory(axonConfiguration.getComponent(QueryBusSpanFactory.class))
+                                 .localSegmentShortCut(true)
```

Comment:
I am not sure what default value should be. Should it be taken from application properties?

Reply:
Adding something to the properties file for this would be very nice indeed! Let me know if you need some guidance with that. :-)

Reply:
I am guessing that this property should be added to `AxonServerConfiguration` ?

Reply:
Ah, sorry for being unclear on this point: yes, the place to add this property would be the `AxonServerConfiguration`.